syntax = "proto3";
package LPMsg;

message Int32Value
{
    int32   value = 1;
}

message VEmpty
{
}

message VString
{
	string	val = 1;
}

message Int32Array
{
	repeated int32	data = 1;
}